components/studio/charts/apps/rstudio/chart/templates/deployment.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.appname }}
+  namespace: {{ .Values.namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      release: {{ $.Release.Name }}
+      app: rstudio
+      project: {{ $.Values.project.slug }}
+      type: app
+      pod: rstudio
+  template:
+    metadata:
+      annotations:
+        prometheus.io/scrape: "true"
+        prometheus.io/path: /metrics
+        # prometheus.io/port: "8501"
+      labels:
+        release: {{ $.Release.Name }}
+        app: rstudio
+        project: {{ $.Values.project.slug }}
+        type: app
+        pod: rstudio
+    spec:
+    {{ if .Values.app.gpu.enabled }}
+      affinity:
+        {{ .Values.app.affinity | toYaml | nindent 8 | trim }}
+    {{ end }}
+    {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      containers:
+      - name: rstudio
```

Comment:
If possible, you want to set up a `readinessProbe` as well, the easiest is probably to just check if the port gives any response:
```
          readinessProbe:
            httpGet:
              path: /
              scheme: HTTP
              port: 8787  # or backend, if you follow my name suggestion in a different comment
            periodSeconds: 20
            timeoutSeconds: 5
            successThreshold: 1
            failureThreshold: 6
```